contrib/zip/example/main.cc

```diff
@@ -0,0 +1,159 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <fcntl.h>
+#include <unistd.h>
+
+#include <cstdlib>
+#include <fstream>
+#include <iostream>
+#include <string>
+#include <vector>
+
+#include "absl/flags/flag.h"
+#include "absl/flags/parse.h"
+#include "contrib/zip/sandboxed.h"
+#include "contrib/zip/utils/utils_zip.h"
+
+ABSL_FLAG(bool, list, false, "list files");
+ABSL_FLAG(std::string, unzip, "", "unzip");
+ABSL_FLAG(std::string, add_file, "", "add file");
+ABSL_FLAG(std::string, delete, "", "delete");
+
+static absl::Status ListFiles(LibZip& zip) {
+  SAPI_ASSIGN_OR_RETURN(int64_t num, zip.GetNumberEntries());
+  for (uint64_t i = 0; i < num; i++) {
+    SAPI_ASSIGN_OR_RETURN(std::string name, zip.GetName(i));
+    std::cout << name << "\n";
+  }
+
+  return absl::OkStatus();
+}
+
+static absl::Status Unzip(LibZip& zip, std::string filename) {
```

Comment:
This should be called `UnzipToStdout` or similar.

Reply:
Fixed.